packages/pangea-node-sdk/src/services/authn/client/index.ts

```diff
@@ -0,0 +1,15 @@
+import BaseService from "../../base";
+import PangeaConfig from "../../../config";
+
+import AuthNClientSession from "./session";
+
+export default class AuthNClient extends BaseService {
+  session: AuthNClientSession;
+
+  constructor(token: string, config: PangeaConfig) {
+    super("authnclient", token, config);
```

Comment:
all BaseService should be initialized with "authn". This serviceName is used to build URL request and all they are related to authn.pangea.cloud.